### PR TITLE
Implement the CLI to support starting covalent with/without dask as well as an easy way to quickly parse the logs

### PR DESCRIPTION
**Pull request by venkatBala**
- [ ] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

File: covalent_dispatcher/_cli/service.py [hunk 1]
```diff
@@ -325,3 +336,20 @@ def purge() -> None:
     cm.purge_config()
 
     click.echo("Covalent server files have been purged.")
+
+
+@click.command()
+def logs() -> None:
+    """
+    Retrive Covalent logs from file
```
Comment (cjao) [inline covalent_dispatcher/_cli/service.py:344]: "Show Covalent server logs"

File: covalent_dispatcher/_cli/service.py [hunk 2]
```diff
@@ -236,13 +244,16 @@ def _graceful_shutdown(pidfile: str) -> None:
     help="Server port number.",
 )
 @click.option("-d", "--develop", is_flag=True, help="Start the server in developer mode.")
+@click.option("--no-cluster", is_flag=True, help="Start the server without Dask")
```
Comment (cjao) [inline covalent_dispatcher/_cli/service.py:247]: We should mention that this would default the executor to "local".

File: covalent_ui/app.py [hunk 3]
```diff
@@ -172,6 +177,7 @@ def serve(path):
         action="store_true",
         help="Start the server in developer mode.",
     )
+    ap.add_argument("--no-cluster", required=False, help="Start Covalent server without Dask")
```
Comment (cjao) [inline covalent_ui/app.py:180]: We should mention that this defaults the executor to "local".

**Review by cjao — CHANGES_REQUESTED** [review]

File: covalent_dispatcher/_cli/service.py (hunk 2 above)
Comment (venkatBala) [reply covalent_dispatcher/_cli/service.py]: Updated the help menu

**Pull request merged**